doc/sphinx/source/conf.py

```diff
@@ -256,7 +256,7 @@
 # One entry per manual page. List of tuples
 # (source start file, name, description, authors, manual section).
 man_pages = [
-    ('index', 'chpldoc', u'chpldoc Documentation',
+    ('index', 'chapel', u'Chapel Documentation',
```

Comment:
This sphinx `conf.py` plays no role in building the `chpldoc` man page. The string `chpldoc` being here was just an artifact of our documentation originating from a `chpldoc`-generated sphinx project.